modules/swagger-codegen/src/main/resources/php/api.mustache

```diff
@@ -103,9 +104,10 @@ use \{{invokerPackage}}\ObjectSerializer;
      * Operation {{{operationId}}}WithHttpInfo
      *
      * {{{summary}}}.
-     *
-  {{#allParams}}   * @param {{dataType}} ${{paramName}} {{description}} {{#required}}(required){{/required}}{{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
-  {{/allParams}}   *
+     */
+  {{#allParams}} //  * @param {{dataType}} ${{paramName}} {{description}} {{#required}}(required){{/required}}{{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
+  {{/allParams}}   
+    /**
```

Comment:
@ePaul Thank you!  The intent for to force it as single line, though I am very open to suggestions on modification here.